frontend/public/components/cluster-settings/oauth.tsx

```diff
@@ -47,15 +47,6 @@ const IdentityProviders: React.FC<IdentityProvidersProps> = ({ identityProviders
   );
 };
 
-// t('public~Basic Authentication')
-// t('public~GitHub')
-// t('public~GitLab')
-// t('public~Google')
-// t('public~HTPasswd')
-// t('public~Keystone')
-// t('public~LDAP')
-// t('public~OpenID Connect')
-// t('public~Request Header')
```

Comment:
@rhamilto @rebeccaalpert i don't think we need the above any more.

Reply:
You're totally right; we don't need these anymore.